installer/install.sh

```diff
@@ -1,7 +1,16 @@
 #!/bin/sh
 
-#  Copyright (C) Marvell Inc
+#  Copyright (C) 2020      Marvell Inc
+#  Copyright (C) 2014-2015 Curt Brune <curt@cumulusnetworks.com>
```

Comment:
Do you think it's better to move create_partition & mount_partition methods in the default_platform.conf into a different file which is specific to x86?

Also, can you briefly summarize the refactoring you did in the installer folder in the PR description?

Reply:
Currently SONiC makes an assumption that all platforms that are amd64: 

1. Support grub
2. Are architect-ed in such a way that these mount / unmount functions will work 

Revoking this assumption would mean copying these functions to the platform-specific `platform.conf` code of many other platforms which would involve a fair amount of repeated code and also involve modifying many non-mellanox platform files. 

Generally we try to avoid modifying platform-specific code outside of Mellanox as that is heavily scrutinized in the community. 

With this we include the create_partition and mount_partition in the `default_platform.conf` file as that configuration is currently considered the "default" platform in SONiC. Any other platforms can override any one or more of these functions as needed.